docs/release-notes/1.10.3.md

```diff
@@ -6,8 +6,12 @@
 ```{rubric} Docs
 ```
 
-```{rubric} Bug fixes
+```{rubric} Bug
```

Comment:
:laughing: you might want to undo this line change :smile:

Reply:
oh no!
yes haha